### DISCUSSION TITLE
Tempo Search Autocomplete Nonsense Values

### DISCUSSION THREAD

**Discussion by GabrielMulcahy** [grafana/tempo#3295] [Q&A]
I am succesfully searching for traces on Tempo using Grafana. All my trace resources have at least these three labels `service`, `environment`, `instance`. In the dropdowns for these labels all the correct values show up *until* I restrict by one, then all the remaining resource values change to seemingly random numbers instead (but still the correct number of options).

This can't be to do with load as this is happening in my test environment with only one resource and my production environment (with many). 

There are no error logs in my test environment whatsoever, so it isn't logging anything to do with this issue.

`instance` dropdown working correctly with no other restrictions:
![image](https://github.com/grafana/tempo/assets/83175003/08cc6899-a8fe-4d3a-9aa7-98e77eb21a6c)

`instance` dropdown now showing a random number since restricting by `environment=TEST`:
![image](https://github.com/grafana/tempo/assets/83175003/adf82c0e-a4da-44ef-a32e-5259c49002aa)

**Comment by mapno** [comment]
Hi! I'm going to close this discussion in favor of https://github.com/grafana/tempo/issues/3297. Please re-open if this one is different.
